cmd/helm/helm.go

```diff
@@ -30,6 +31,12 @@ func init() {
 	addCommands(cmds()...)
 }
 
+// Debug indicates whether the process is in debug mode.
+//
+// This is set at app start-up time, based on the presence of the --debug
+// flag.
+var Debug bool
```

Comment:
I check it in `before()` now so that all of the commands can access it. Otherwise every command needs to parse the flag. Didn't need to make it public, though. I'll switch that to lowercase.